x-pack/plugins/observability/public/pages/alerts/alerts.tsx

```diff
@@ -130,6 +133,7 @@ function InternalAlertsPage() {
       const response = await loadRuleAggregations({
         http,
         typesFilter: filteredRuleTypes,
+        filterConsumers: observabilityRuleCreationValidConsumers,
```

Comment:
Why `filterConsumers` should be provided here?

Reply:
Yes, we should not use anymore this `const` I change it back to `observabilityAlertFeatureIds`

Reply:
It is because now we have some rule types that can be common between observability and stack management, so we need an extra level of checking for consumers.